package.json

```diff
@@ -11,7 +11,8 @@
     "lint": "eslint --ext .ts server/",
     "lint-fix": "eslint --ext .ts server/ --fix && cd client && npm run lint-fix",
     "lint:client": "cd client && npm run lint",
-    "build:db":"NODE_ENV=dev ts-node server/database/config/build.ts"
+    "build:db": "NODE_ENV=dev ts-node server/database/config/build.ts",
+    "db:fake": "NODE_ENV=dev ts-node server/database/fakeData/index.ts --resolveJsonModule"
```

Comment:
You can add a short explanation in the description of this script and for `--resolveJsonModule`.